app/graphql/types/cash_flow_type.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Types
+  class CashFlowType < Types::BaseObject
+    # field :username, String, null: true
+    # field :years, [Types::YearType], null: true
```

Comment:
I now see the YearType custom, I recommend deleting this reference and other 2 files;
we can always reference the previous commit if we want to explore, but as for data going into main, should be removed